views/info/filters.pug

```diff
@@ -0,0 +1,29 @@
+extends ../layout
+
+block content
+  include ../flash
+  br
+  h4 Filter Syntax Guide
+  br
+  dl.row
+    dt.col-3 General
+    dd.col-9
+      p You can combine any number of filters together, using AND, or OR.
+      p t:instant or t:sorcery //cards that are either instants or sorceries
+      p t:instant t:tribal //cards that are both instants and tribal
+      p You can also use quotes to ignore spaces.
+      p goblin blood //cards whose names contain both blood and goblin
+      p "goblin blood" //cards whose names contain goblin blood
```

Comment:
Are these comments user-visible? If so, they could use some clarification.

Reply:
@omniczech Said he would fix this file after I pushed it